### ISSUE TITLE
Accessibility of Buttons?

### ISSUE THREAD

**Issue by kylejrp** [kognise/water.css#228]
#225 fixed the buttons colors being indistinguishable in tables, but when I checked the color contrasts, they're still a little lacking.

I'm thinking that we need to make the color contrast even stronger. Here's the contrast ratio I'm seeing in each theme/background combination using a [contrast checker](https://webaim.org/resources/contrastchecker/):
![image](https://user-images.githubusercontent.com/6742479/95543685-12d76e80-09ae-11eb-8345-5538e830de0d.png) ![image](https://user-images.githubusercontent.com/6742479/95543690-1539c880-09ae-11eb-944f-c482b406523e.png)

| Background | Light Theme | Dark Theme |
|---------------|--------------|---------------|
| Light             |  1.14:1        |    1.16:1      |
| Dark             |   1.07:1        |    1.06:1     |

It seems like a color contrast ratio of [3:1 is recommended for accessibility with these elements](https://www.w3.org/TR/WCAG21/#non-text-contrast).

It seems like this contrast might be lacking on `<button>` elements on the normal backgrounds too.

I'm not a designer, so I would love if someone with a mind for colors would be able to provide some guidance here. 😅

**Comment by HimanshuPandey2002** [comment]
Hey can I work on this issue, I dont have much experience in Open source but I want to contribute so i think this issue would be perfect for me as I work on frontend a lot, you can see my work [here](https://github.com/HimanshuPandey2002l)

**Comment by kylejrp** [comment]
Awesome, go for it @HimanshuPandey2002! Feel free to open up a WIP pull request if you need feedback and feel free to ask any questions if you need help. You might want to review the changes made in #225 as they made some changes to how buttons work in tables.

Good luck!

**Comment by imsky003** [comment]
hey @kylejrp  i can make light button in your dark mode, if you are intrested please let me do pull request so that i can gain more pull request in Hacktoberfest event

**Comment by HimanshuPandey2002** [comment]
Thanks @kylejrp Im on it.
